modules/op-release-notes-1-16.adoc

```diff
@@ -0,0 +1,594 @@
+          spec:
+            template:
+              spec:
+                containers:
+                - name: tekton-chains-controller
+                  volumeMounts:
+                  - mountPath: /tmp/mongo-url
+                    name: mongo-url
+                volumes:
+                -  name: mongo-url
+                   secret:
+                    secretName: mongo-url
+# ...
+----
+
+* With this update, when using KMS signing in {tekton-chains}, instead of providing the KMS authentication token value directly in the configuration, you can provide the token value as a secret by using the `signers.kms.auth.token-path` setting.  
++
+.Example of creating a KMS token secret
+[source,terminal]
+----
+$ oc create secret generic kms-secrets -n tekton-chains \
+  --from-file=KMS_AUTH_TOKEN=/home/user/KMS_AUTH_TOKEN
+----
++
+.Example configuration of the KMS token value using a secret
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+# ...
+  chain:
+    disabled: false
+    signers.kms.auth.token-path: /etc/kms-secrets/KMS_AUTH_TOKEN
+    options:
+      deployments:
+        tekton-chains-controller:
+          spec:
+            template:
+              spec:
+                containers:
+                - name: tekton-chains-controller
+                  volumeMounts:
+                  - mountPath: /etc/kms-secrets
+                    name: kms-secrets
+                volumes:
+                -  name: kms-secrets
+                   secret:
+                    secretName: kms-secrets
+# ...
+----
+
+* With this update, you can configure a list of namespaces as an argument to the {tekton-chains} controller. If you provide this list, {tekton-chains} watches pipeline runs and task runs only in the specified namespaces. If you do not provide this list, {tekton-chains} watches pipeline runs and task runs in all namespaces.
++
+.Example configuration for watching only the `dev` and `test` namespaces
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+  chain:
+    disabled: false
+    options:
+      deployments:
+        tekton-chains-controller:
+          spec:
+            template:
+              spec:
+                containers:
+                - args:
+                  - --namespace=dev, test 
+                  name: tekton-chains-controller
+----
+
+[id="tekton-results-new-features-1-16_{context}"]
+=== {tekton-results}
+
+* Before this update, {tekton-results} used the `v1beta1` API format to store `TaskRun` and `PipelineRun` object records. With this update, {tekton-results} uses the `v1` API format to store `TaskRun` and `PipelineRun` object records.
+
+* With this update, {tekton-results} now supports fetching forwarded logs from third party logging APIs. You can enable the logging API through the `TektonResult` CR by setting the `logs_api` to `true` and `logs_type` to `Loki`.
```

Comment:
I suggest moving this one under the next one just to keep the two RNs about API versions together